### ISSUE TITLE
The `log` option's types are mismatched

### ISSUE THREAD

**Issue by adamalston** [bahmutov/cypress-cdp#99]
Per the code, `log` can be set to a boolean value in the `CDP` command:

https://github.com/bahmutov/cypress-cdp/blob/3cbd2061cadd2d2764f999d3e238fb1c21c25e0f/src/index.js#L21

However, `LogConfig` is not a boolean type:

https://github.com/bahmutov/cypress-cdp/blob/3cbd2061cadd2d2764f999d3e238fb1c21c25e0f/src/index.d.ts#L28

Cypress `LogConfig` type:

https://github.com/cypress-io/cypress/blob/1aad7ba7d1b28d2ea920cdc65a54af74ad470643/cli/types/cypress.d.ts#L6447-#L6466

**Comment by Capocaccia** [comment]
Agreed. As a result it makes it really difficult to use the hasEventListeners function with a type object provided.

**Comment by joemaffei** [comment]
The README shows that it's possible to use an object with just the `type` property set: https://github.com/bahmutov/cypress-cdp/blob/main/README.md?plain=1#L78

For now, this is my workaround:

```ts
cy.hasEventListeners('form',  {
  type: 'submit',
  // @ts-ignore https://github.com/bahmutov/cypress-cdp/issues/99
  log: false,
});
```

**Comment by MattWeinstein** [comment]
Since merging [this PR](https://github.com/bahmutov/cypress-cdp/pull/100), `log` has been made optional which partially resolves this issue.

**Comment by Capocaccia** [comment]
Agreed @MattWeinstein.
